gateway/gateway.go

```diff
@@ -104,6 +103,7 @@ type deviceInfo struct {
 	AppSKey  string  `json:"app_skey"`
 	NwkSKey  string  `json:"nwk_skey"`
 	FCntDown *uint16 `json:"fcnt_down"`
+	NodeName string  `json:"node_name"`
```

Comment:
A little confused why we're adding this as a singular string - couldn't there be multiple nodes per gateway?

Reply:
We have multiple devices per gateway, and each device only has one name technically